### PR TITLE
fix bug: reverse lat & lng in NavigationUIActivity sample

### PR DESCRIPTION
**Pull request by tallongsun**
Fix bug: lat & lng is reversed in NavigationUIActivity sample

### CODE REVIEW

**Comment by Fabi755** [comment]
Oh, thank you for the quick fix!

Can you clean up the whitespaces, for a more consistent look?! 😄

File: app/src/main/java/com/mapbox/services/android/navigation/testapp/NavigationUIActivity.kt [hunk 1]
```diff
@@ -71,7 +71,7 @@ class NavigationUIActivity :
                 val options = NavigationLauncherOptions.builder()
                     .directionsRoute(route)
                     .shouldSimulateRoute(simulateRoute)
-                    .initialMapCameraPosition(CameraPosition.Builder().target(LatLng(userLocation.longitude, userLocation.latitude)).build())
+                    .initialMapCameraPosition(CameraPosition.Builder().target(LatLng( userLocation.latitude,userLocation.longitude)).build())
```
Comment (Fabi755) [inline app/src/main/java/com/mapbox/services/android/navigation/testapp/NavigationUIActivity.kt:74]:
```suggestion
                    .initialMapCameraPosition(CameraPosition.Builder().target(LatLng(userLocation.latitude, userLocation.longitude)).build())
```

**Comment by Fabi755** [comment]
I see that I can do by myself. Already done!

Thanks again! :-)

**Review by Fabi755 — APPROVED** [review]

**Review by boldtrn — APPROVED** [review]
Thanks, LGTM 👍

**Comment by tallongsun** [comment]
@Fabi755 @boldtrn  Thanks for review.
Would like to know how this PR can be merged?

**Pull request merged**